### ISSUE TITLE
Error: "Can't set headers after they are sent"

### ISSUE THREAD

**Issue by wrought** [sudoroom/sudo-humans#59]
I've seen this several times -- both in trying to browse to `/~user` in my local development environent, and also when we have an issue with a user editing their account information #56 What does it mean?

```
http.js:689
    throw new Error('Can\'t set headers after they are sent.');
          ^
Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (http.js:689:11)
    at /home/wrought/projects/sudo-humans/lib/layout.js:16:17
    at error (/home/wrought/projects/sudo-humans/bin/cmd.js:268:11)
    at /home/wrought/projects/sudo-humans/routes/payment.js:57:31
    at null._onTimeout (/home/wrought/projects/sudo-humans/node_modules/stripe/lib/StripeResource.js:86:35)
    at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)
```

**Comment by wrought** [comment]
In my local environment, this error went away after a few minutes. I was able to browse to `/~user` without this error. No explanation so far.

**Comment by okdistribute** [comment]
This happens when the server writes to the response twice.

**Comment by Juul** [comment]
Fixed by 5cb54cf16124de515c8e4a686dc2b0d73f4f9102
